src/atoms/Space/__tests__/Space.web.test.js

```diff
@@ -1,39 +1,45 @@
 import React from 'react';
-import { render } from '@testing-library/react';
 import View from '../../View';
 import Space from '../index';
+import { renderWithTheme } from '../../../_helpers/testing';
 
 beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
 afterAll(() => jest.restoreAllMocks());
 
-describe('Renders <Space /> correctly', () => {
-  it('should apply padding to child component', () => {
-    const { container } = render(
-      <Space padding={[1, 2]}>
-        <View />
-      </Space>,
-    );
-    expect(container).toMatchSnapshot();
-  });
-
-  it('should apply margin to child component', () => {
-    const { container } = render(
-      <Space margin={[5, 5]}>
-        <View />
-      </Space>,
-    );
-    expect(container).toMatchSnapshot();
+describe('<Space />', () => {
+  describe('padding', () => {
+    it('should apply Padding to child component', () => {
+      const displayText = 'Flex Item';
+      const { container } = renderWithTheme(
+        <Space padding={[1, 2]}>
+          <View>{displayText}</View>
+        </Space>,
+      );
+      expect(container).toMatchSnapshot();
+    });
   });
-
-  it('should throw error when more than one nodes are passed as children', () => {
-    const expectedErrorMessage = 'Expected a single child for Space component';
-    expect(() =>
-      render(
+  describe('margin', () => {
+    it('should apply Margin to child component', () => {
+      const displayText = 'Flex Item';
+      const { container } = renderWithTheme(
         <Space margin={[5, 5]}>
-          <View />
-          <View />
+          <View>{displayText}</View>
         </Space>,
-      ),
-    ).toThrow(expectedErrorMessage);
+      );
+      expect(container).toMatchSnapshot();
+    });
+  });
+  describe('error', () => {
+    it('should throw error when more than one nodes are passed as children', () => {
```

Comment:
`'should throw an error when more than one nodes are passed as children'`

Reply:
made this as `throws...`  . same as the screenshot in the thread.